truss/env.go

```diff
@@ -0,0 +1,30 @@
+package truss
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// EnvInput input
+type EnvInput struct {
+	Env         string
+	Kubeconfigs map[string]interface{}
+	KubeDir     string
+}
+
+// Env returns string to eval that configures shell environment variables
+func Env(input *EnvInput) (string, error) {
+	buffer := bytes.NewBuffer(make([]byte, 0))
+	kubeconfigName := input.Kubeconfigs[input.Env]
+
+	if kubeconfigName == nil {
+		return "", fmt.Errorf("No kubeconfig found for env %s", input.Env)
+	}
+
+	kubeconfig := fmt.Sprintf("%s%s", input.KubeDir, kubeconfigName)
+	buffer.WriteString(fmt.Sprintf("export KUBECONFIG=%s\n", kubeconfig))
```

Comment:
thoughts on returning these as a struct and having the cli handing the string format?

Reply:
That's not a bad idea, let me think on that.